src/System.Runtime/tests/System/DecimalTests.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace System.Tests
 {
+    [Collection("CultureTestCollection")]
```

Comment:
You are right, I just tested it inside a loop. Makes sense as it lives in the CurrentThread... Thanks!